pkg/types/baremetal/validation/platform.go

```diff
@@ -1,13 +1,17 @@
 package validation
 
 import (
+	"fmt"
+	"net"
+
+	"github.com/openshift/installer/pkg/types"
 	"github.com/openshift/installer/pkg/types/baremetal"
 	"github.com/openshift/installer/pkg/validate"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 )
 
 // ValidatePlatform checks that the specified platform is valid.
-func ValidatePlatform(p *baremetal.Platform, fldPath *field.Path) field.ErrorList {
+func ValidatePlatform(p *baremetal.Platform, fldPath *field.Path, n *types.Networking) field.ErrorList {
```

Comment:
can be keep this new argument before the fldPath... ?

Reply:
Sure.